app/src/main/java/com/example/minutejournal/DTO/EntriesDTO.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.minutejournal.DTO
```

Comment:
Even though DTO is an acronym, package names in Java/Kotlin are all lowercase, so I'd lowercase dto in the package name.